projects/1.16/assets/evilcraft/evilcraft/lang/zh_cn.json

```
@@ -1 +1,765 @@
+  "item.evilcraft.flesh_rejuvenated": "再生鲜肉",
+  "item.evilcraft.flesh_rejuvenated.info": "无限的食物来源",
+  "item.evilcraft.lightning_grenade": "雷击手榴弹",
+  "item.evilcraft.redstone_grenade": "红石脉冲弹",
+  "item.evilcraft.bucket_blood": "血液桶",
+  "item.evilcraft.bucket_poison": "毒液桶",
+  "item.evilcraft.blood_extractor": "血液提取器",
+  "item.evilcraft.blood_extractor.info": "放在背包，杀死生物时会收集血液。\\nShift+右键切换自动补给。",
+  "item.evilcraft.blood_extractor.info.auto_supply": "自动补给: %s",
+  "item.evilcraft.dark_gem": "暗黑宝石",
+  "item.evilcraft.dark_stick": "暗黑木棍",
+  "item.evilcraft.weather_container": "天气容器",
+  "item.evilcraft.blood_pearl_of_teleportation": "血能末影珍珠",
+  "item.evilcraft.broom": "扫帚",
+  "_item.evilcraft.broom.info": "I can fly!",
+  "item.evilcraft.broom_part": "扫帚部件",
+  "item.evilcraft.hardened_blood_shard": "干血碎片",
+  "item.evilcraft.hardened_blood_shard.info": "在血块上使用打火石可以获得。",
+  "item.evilcraft.blood_infusion_core": "血能注入核心",
+  "item.evilcraft.dark_power_gem": "暗黑充能宝石",
+  "item.evilcraft.dark_power_gem.info": "将暗黑宝石投入至少5格的流动血液中，或者用血能注入器将血能注入暗黑宝石",
+  "item.evilcraft.blood_container.bloodCell": "血瓶",
+  "item.evilcraft.blood_container.bloodCan": "血罐",
+  "item.evilcraft.blood_container.bloodBasin": "血盆",
+  "item.evilcraft.blood_container.creativeBloodContainer": "创造模式血箱",
+  "item.evilcraft.blood_container.info": "不推荐使用！请合成转换为暗黑液体罐。",
+  "item.evilcraft.blood_container.info.autoSupply": "自动补给模式:%s",
+  "item.evilcraft.poison_sac": "毒囊",
+  "item.evilcraft.werewolf_fur": "狼人皮",
+  "item.evilcraft.blook": "血之书",
+  "item.evilcraft.blook.info": "放入净化器中可以吸收物品的附魔。",
+  "item.evilcraft.potentia_sphere": "神秘球",
+  "item.evilcraft.potentia_sphere.info": "注入血液会变成末影珍珠。",
+  "item.evilcraft.inverted_potentia": "暗黑能量球",
+  "item.evilcraft.inverted_potentia.info": "用闪电可以充能。",
+  "item.evilcraft.inverted_potentia_empowered": "充能的暗黑能量球",
+  "item.evilcraft.inverted_potentia_empowered.info": "被闪电击中。",
+  "item.evilcraft.mace_of_distortion": "立场扭曲锤",
+  "item.evilcraft.kineticator": "磁场魔杖",
+  "item.evilcraft.kineticator.info": " Shift+右键开启魔杖。\\n右键改变作用面积。",
+  "item.evilcraft.kineticator.info.attraction": "吸引: %s",
+  "item.evilcraft.kineticator.info.area": "吸引范围: %s",
+  "item.evilcraft.kineticator_repelling": "反磁场魔杖",
+  "item.evilcraft.kineticator_repelling.info": " Shift+右键开启魔杖。\\n右键改变作用面积。.",
+  "item.evilcraft.kineticator_repelling.info.attraction": "排斥: %s",
+  "item.evilcraft.kineticator_repelling.info.area": "排斥范围: %s",
+  "item.evilcraft.vengeance_ring": "复仇指环",
+  "item.evilcraft.vengeance_ring.info": "可能会吸引或召唤复仇之魂 \\n shift +右键开关",
+  "item.evilcraft.vengeance_ring.info.status": "状态: %s",
+  "item.evilcraft.vengeance_focus": "复仇之焦",
+  "item.evilcraft.vengeance_focus.info": "右键灵魂施加封印，会使附近的空灵魂箱会吸收灵魂。",
+  "item.evilcraft.vengeance_pickaxe": "复仇之镐",
+  "item.evilcraft.vengeance_pickaxe.info": "可能会召唤复仇之魂",
+  "item.evilcraft.burning_gem_stone": "灼烧暗黑宝石块",
+  "item.evilcraft.burning_gem_stone.info": "将复仇之魂的伤害转换为饥饿.",

```

|Feedback Priority|
|---|
|⛰ Mountain / **Blocking and requires immediate action**|

```suggestion
  "item.evilcraft.burning_gem_stone.info": "被动地将复仇之魂的伤害转换为饥饿值消耗。",
```